js/scroll-to-top.js

```diff
@@ -0,0 +1,24 @@
+namespacer('bcpl');
+
+bcpl.scrollToTop = (($) => {
+	const backToTopButtonSelector = '#scroll-to-top';
+	const bodyHtmlSelector = 'body, html';
+	const scrollSpeed = 250;
+	const topScrollPosition = 0;
+
+	const scrollToTopHandler = () => {
+		$(bodyHtmlSelector).animate({
+			scrollTop: topScrollPosition
+		}, scrollSpeed);
+	};
+
+	const init = () => {
+		$(document).on('click', backToTopButtonSelector, scrollToTopHandler);
+	};
+
+	return {
+		init
+	};
+})(jQuery);
+
+$(() => bcpl.scrollToTop.init());
```

Comment:
Do we want to make this into a small module or package for reuse? You could pass in the selectors, speed and position as options?